examples.md

```diff
@@ -456,7 +456,9 @@ When dependencies are installed later in the workflow, we must specify the same
       ~/.cargo/registry
       ~/.cargo/git
       target
-    key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+    key: cargo-${{ hashFiles('**/Cargo.lock') }}
+    restore-keys: |
+      cargo-
```

Comment:
This has a disadvantage: without pruning, the crates cached in the `registry` and `git` subdirectories will accumulate by being handed down from earlier cache states to the fresh one.